weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/context.tsx

```diff
@@ -51,6 +51,7 @@ const WFDataModelFromTraceServerCallsOnlyProvider: FC = ({children}) => {
     return {
       useCall: tsWFDataModelHooks.useCall,
       useCalls: tsWFDataModelHooks.useCalls,
+      useCallsDelete: tsWFDataModelHooks.useCallsDelete,
```

Comment:
nit/question: `useCallsDelete` follows a slightly different pattern (it returns a function). I wonder if there is a naming convention that could imply this pattern difference?